crates/proto/src/rr/rdata/tlsa.rs

```diff
@@ -92,6 +92,7 @@ pub enum CertUsage {
     ///       the certificate might or might not have the basicConstraints
     ///       extension present.
     /// ```
+    #[cfg_attr(feature = "serde", serde(rename = "PKIX-TA"))]
```

Comment:
IMO for these cases where the differences is more than just punctuation/capitalization, we should probably align the variant identifier more with the spec, like renaming this to `PkixTa`.

@marcus0x62 thoughts?